Jenkinsfile

```diff
@@ -30,7 +30,7 @@ properties(
     ]
 )
 final def oses = ['linux':'ubuntu && maven', 'windows':'windows-he']
-final def mavens = env.BRANCH_NAME == 'master' ? ['3.6.x', '3.2.x'] : ['3.2.x']
+final def mavens = env.BRANCH_NAME == 'master' ? ['3.6.x', '3.8.x'] : ['3.8.x']
```

Comment:
This change is not acceptable. We want to and we will accept the range Maven 3.2+.

Reply:
current used Maven api is 3.2.5 ... so we need check with it.
3.8.x is ok

Reply:
@slawekjaranowski sounds a good explanation to change that

Reply:
fixed